internal/receiver/scriptedinputsreceiver/procpipe/commander.go

```diff
@@ -0,0 +1,207 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procpipe
+
+import (
+	"context"
+	"crypto/sha512"
+	"encoding/hex"
+	"io"
+	"os"
+	"os/exec"
+	"sync/atomic"
+	"syscall"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+const LinuxScriptsLocation = "/etc/otel/collector/scripts/"
```

Comment:
I'd think its preferable for all of the scripts to be embedded in the executable instead of relying on the host fs: https://pkg.go.dev/embed

Reply:
This creates an unnecessary packaging requirement and opens up the collector to rce. There is no functional requirement for this to be the case and is a security concern.

Reply:
@weliasz feel free to deviate from the initial design doc where it makes sense.

@rmfitzpatrick you may want to provide additional details about how you think the alternate approach you propose should work.

Reply:
@rmfitzpatrick Could you hint me on how I could execute script from go:embed using exec.CommandContext?

Reply:
I've figured out how to do it.